cobra/cmd/license_lgpl.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+func initLgpl() {
+	Licenses["lgpl"] = License{
+		Name:            "GNU Lesser General Public License",
+		PossibleMatches: []string{"lgpl", "lesser gpl", "gnu lgpl"},
```

Comment:
As you can see in the archive.org the text of the license has changed over time at the same URL: https://web-beta.archive.org/web/*/https://www.gnu.org/licenses/lgpl.txt    and the LGPL states in section 6: https://www.gnu.org/licenses/lgpl-3.0.html#section6
`If the Library as you received it does not specify a version number of the GNU Lesser General Public License, you may choose any version of the GNU Lesser General Public License ever published by the Free Software Foundation.` and this applies to the LGPL 2.0, 2.1 and 3.0. So a plain LGPL can be any version 2 or later and a  Less GPL can be any version 2.1 or later.